src/main/java/duke/core/TypoCorrector.java

```diff
@@ -20,13 +20,13 @@
 
     //Sets of "Dictionaries" for the command keyword, categorised by number of keywords contain in a supported commands.
     private static final ArrayList<String> oneKeywordCommand = new ArrayList<String>(
-            Arrays.asList("bye"));
+        Arrays.asList("bye", "duke", "help"));
     private static final ArrayList<String> twoKeywordsCommands = new ArrayList<String>(
-            Arrays.asList("list patients", "list tasks"));
+        Arrays.asList("list patients", "list tasks"));
     private static final ArrayList<String> otherCommands = new ArrayList<String>(
-            Arrays.asList("update patient", "update task",
-                    "delete patient", "delete task", "add task", "add patient",
-                    "assign by", "find patient", "find task"));
+        Arrays.asList("update patient", "update task",
+            "delete patient", "delete task", "delete patienttask", "add task", "add patient",
+            "find patient", "find task", "assign standardtask", "assign eventtask"));
```

Comment:
Same brought up in my earlier comment; no need to squish together `standardtask`. `assign standard task` and `assign event task` should be expected to work.